understanding/20/images-of-text-no-exception.html

```diff
@@ -12,11 +12,12 @@ <h1>Understanding Images of Text (No Exception)</h1>
    <section id="brief">
       <h2>In brief</h2>
       <dl>
-         <dt>Objective</dt><dd>Users can always adjust how text is presented</dd>
-         <dt>Author task</dt><dd>Do not use pictures of text unless there is no other way to present information</dd>
+         <dt>Goal</dt><dd>Users can always adjust how text is presented.</dd>
+         <dt>What to do</dt><dd>Do not use pictures of text unless there is no other way to present information.</dd>
+         <dt>Why it's important</dt><dd>People cannot alter how text looks in images.</dd> 
```

Comment:
@rainbreaw suggests
```suggestion
         <dt>Why it's important</dt><dd>People cannot alter how text looks in images, and at the time of publishing, assistive technology is inconsistent in its ability to read text in images aloud.</dd> 
```

Reply:
I don't think this is an appropriate addition, especially if you look at the AA wording of Images of text:
> If the technologies being used can achieve the visual presentation...

These SCs are predominantly about visual perception, and are targetted at the ability to manipulate the presentation, so talking about ATs reading text aloud seems off target. Note also that it is possible to meet 1.1.1  Non-text Content here (i.e., there is ALT that provides the same text) yet still fail these Images of Text criteria.
Finally, I think we should avoid 'timestamping' our guidance in this way.

Reply:
Mbgower -- you missed the point.  
You are looking at the exception 0-- not the purpose of the rule.

The purpose of the rule is to ensure that all text is programmatically readable  (so you can change to voice or braille or simpler text) 

The exception is --- If there is NO WAY to make the text programmatically present the way you need it to be -- then you can use a picture of the text (and alt text).   Otherwise you must use real text and format it so that it can be read by assistive technologies.